autometacal/python/moments.py

```diff
@@ -0,0 +1,26 @@
+from .tf_ngmix.moments import get_moments
+from .tf_ngmix.gmix import create_gmix,  fwhm_to_T
+from .tf_ngmix.pixels import make_pixels
+import tensorflow as tf
+
+def gaussian_moments(images, scale, fwhm, **kwargs):
```

Comment:
This is the user-level API, it needs to be properly documented with docstrings. Also, the name is misleading, it's not actually returning moments but ellipticities.